modules/programs/man.nix

```
@@ -39,7 +46,7 @@ with lib;
   };
 
   config = mkIf config.programs.man.enable {
-    home.packages = [ pkgs.man ];
+    home.packages = [ cfg.package ];
```

```suggestion
    home.packages = [ config.programs.man.package ];
```

Or you can add
``` nix
let
  cfg = config.programs.man;
in
```
at the top.

---

Yep, I missed that!